src/workerd/util/use-perfetto-categories.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <workerd/util/perfetto-tracing.h>
+
+// This header is to be imported by any translation unit (c++ file) that
+// is instrumenting with perfetto traces (e.g. TRACE_EVENT). Header files
+// that include instrumentation but are imported by embedders should instead
+// use PERFETTO_USE_CATEGORY_FROM_NAMESPACE_SCOPED within function or block
```

Comment:
`PERFETTO_USE_CATEGORY_FROM_NAMESPACE_SCOPED` is not defined in the Perfetto Trace SDK.

Reply:
Looks like it's misspelled, and should be `..._CATEGORIES_...`. https://github.com/google/perfetto/blob/453a8a40196bea0033d205a6d0c1a0075aee65b1/include/perfetto/tracing/track_event.h#L220-L223